02-peer-protocol.md

```diff
@@ -935,7 +935,7 @@ updates.
   - MAY send a `commitment_signed` message that only
 alters the fee.
   - MAY send a `commitment_signed` message that doesn't
-change the commitment transaction aside from the new revocation hash
+change the commitment transaction aside from the new payment hash
```

Comment:
Yes.  I agree it would be clearer to say "we can send commitment_signed after sending 1 or mroe updae" but since we are trying to close 1.0, let's defer that.